Website/src/rendering/src/components/Vendors/VendorInformation.tsx

```diff
@@ -1,29 +1,47 @@
 import { ComponentProps } from 'lib/component-props';
 import { Field, RichText } from '@sitecore-jss/sitecore-jss-nextjs';
 import SessionList from '../Sessions/SessionList';
-import { SESSIONS } from '../../models/mock-sessions';
+import { GraphQLSession } from 'src/types/session';
 
 export type VendorInformationProps = ComponentProps & {
   fields: {
-    Description: Field<string>;
+    data: {
+      contextItem: {
+        description: Field<string>;
+        sessions: {
+          targetItems: GraphQLSession[];
+        };
+      };
+    };
   };
 };
 
-const VendorInformation = (props: VendorInformationProps): JSX.Element => (
-  <section className="section information-section">
-    <div className="section__content container">
-      <div className="information-grid">
-        <div className="main-col">
-          <div className="column-title">Vendor history:</div>
-          <RichText className="rich-text" field={props.fields.Description} />
-        </div>
-        <div className="sidebar-col">
-          <div className="column-title">Sessions:</div>
-          <SessionList sessions={SESSIONS} showSpeakers={true} />
+const VendorInformation = (props: VendorInformationProps): JSX.Element => {
+  const SessionListSidebar =
+    props.fields.data.contextItem.sessions.targetItems.length > 0 ? (
+      <>
+        <div className="column-title">Sessions:</div>
+        <SessionList
```

Comment:
SessionList seems to have a bug. The speaker links are not valid. Example: https://jel-pull0250-website.sitecoredemo.com/speakers/undefined

Could you please fix it in this PR as you used `showSpeakers={true}` ?